2018/TPAC.md

```diff
@@ -27,6 +27,12 @@ etc.
 
 ## Agenda items
 
+**Please note: the schedule as described on the w3 website is not correct**
+
+The [TPAC schedule](https://www.w3.org/2018/10/TPAC/schedule.html) says that Thursday and Friday are full days for the WebAssembly Working Group, and there is a two-hour meeting on Tuesday for the Community Group. This is **not** correct.
+
+Instead, Thursday and Friday will be full days of Community Group discussion. You only need to be a member of the Community Group to attend.
```

Comment:
TPAC website says that it's also necessary to pay registration fees (per person per day)

Reply:
Oops, thanks.

Reply:
Clarified here: https://github.com/WebAssembly/meetings/pull/308